Mathlib/Analysis/InnerProductSpace/Orthonormal.lean

```diff
@@ -50,6 +50,16 @@ def Orthonormal (v : ι → E) : Prop :=
 
 variable {𝕜}
 
+@[simp]
+lemma Orthonormal.of_isEmpty [IsEmpty ι] (v : ι → E) : Orthonormal 𝕜 v :=
+  ⟨IsEmpty.elim ‹_›, Subsingleton.pairwise⟩
+
+@[simp]
+lemma orthonormal_vecCons_iff {n : ℕ} {v : E} {vs : Fin n → E} :
+    Orthonormal 𝕜 (Matrix.vecCons v vs) ↔ ‖v‖ = 1 ∧ (∀ i, ⟪v, vs i⟫ = 0) ∧ Orthonormal 𝕜 vs := by
```

Comment:
`v` and `vs` appear once on the LHS and twice on the RHS. Are you sure you want it to be simp?

Reply:
I'm inclined to think this is correct. Sure, the LHS only has one of each, but they're wrapped in `Matrix.vecCons`, so it's really this I think that this the deciding factor.